gasCosts/gasCosts.js

```diff
@@ -110,56 +110,63 @@ contract("All", accounts => {
     });
 
     it("when working with a Task", async () => {
-      await colony.makeTask(SPECIFICATION_HASH, 1);
-      await colony.setTaskDomain(1, 1);
-      await colony.setTaskSkill(1, 7);
-      await colony.setTaskRoleUser(1, EVALUATOR_ROLE, EVALUATOR);
-      await colony.setTaskRoleUser(1, WORKER_ROLE, WORKER);
+      const { logs } = await colony.makeTask(SPECIFICATION_HASH, 1);
+      const taskId = logs[0].args.id.toNumber();
+      await colony.setTaskRoleUser(taskId, EVALUATOR_ROLE, EVALUATOR);
+      await colony.setTaskRoleUser(taskId, WORKER_ROLE, WORKER);
```

Comment:
Can you delay setting the worker and execute the `setTaskDomain` and `setTaskSkill` with just the manager signature? This will reflect on the reported gas prices to show a better average inclusive of this case.